File: mod10/dia_1/exercicio1.js
function sum(number1, number2) {
  return number1 + number2;
};

function myRemove([...arr], number) {
  return arr.filter((arrNumber) => arrNumber !== number)
};

function myRemoveWithoutCopy(arr, number) {
  if (arr.includes(number)) arr.splice(arr.indexOf(number), 1);
  return arr;
};

function fizzBuzz(num) {
  if (typeof num !== 'number') return false;
  if (num % 3 === 0 && num % 5 === 0) return 'fizzbuzz';
  if (num % 3 === 0) return 'fizz';
  if (num % 5 === 0) return 'buzz';
  return num;
};

module.exports = {
  sum,
  myRemove,
  myRemoveWithoutCopy,
  fizzBuzz,
};

File: mod32/dia_3/exercicio_1.py
def fizz_buzz(limit):
    result = []
    for n in limit:
        if (n % 3 == 0 and n % 5 == 0):
            result.append('FizzBuzz')
        elif (n % 3 == 0):
            result.append('Fizz')
        elif (n % 5 == 0):
            result.append('Buzz')
        else:
            result.append(n)
    return result


print(fizz_buzz([1, 2, 3, 4, 5, 6, 7, 8, 9, 10, 11, 12, 13, 14, 15]))
